beta/src/pages/learn/index.md

```diff
@@ -4,716 +4,486 @@ title: Quick Start
-  url: 'https://i.imgur.com/okTpbHhm.jpg',
-  alt: 'A pale wire-like sculpture mounted on concrete wall and descending on the floor. It appears light.'
-}, {
-  name: 'Hippos',
-  artist: 'Taipei Zoo',
-  description: 'The Taipei Zoo commissioned a Hippo Square featuring submerged hippos at play.',
-  url: 'https://i.imgur.com/6o5Vuyu.jpg',
-  alt: 'A group of bronze hippo sculptures emerging from the sett sidewalk as if they were swimming.'
-}];
-```
-
-```css
-h2 { margin-top: 10px; margin-bottom: 0; }
-h3 {
- margin-top: 5px;
- font-weight: normal;
- font-size: 100%;
-}
-img { width: 120px; height: 120px; }
-button {
-  display: block;
-  margin-top: 10px;
-  margin-bottom: 10px;
-}
-```
-
 </Sandpack>
 
-State can hold complex values, too. For example, if you're implementing a form, you can keep an object in state with different fields. The `...` syntax in the below example lets you [create new objects based on existing ones](/learn/updating-objects-in-state).
-
-<Sandpack>
-
-```js
-import { useState } from 'react';
-
-export default function Form() {
-  const [person, setPerson] = useState({
-    firstName: 'Barbara',
-    lastName: 'Hepworth',
-    email: 'bhepworth@sculpture.com'
-  });
-
-  function handleFirstNameChange(e) {
-    setPerson({
-      ...person,
-      firstName: e.target.value
-    });
-  }
+## Responding to events {/*responding-to-events*/}
 
-  function handleLastNameChange(e) {
-    setPerson({
-      ...person,
-      lastName: e.target.value
-    });
-  }
+You can respond to events by declaring event handler functions inside your components:
 
-  function handleEmailChange(e) {
-    setPerson({
-      ...person,
-      email: e.target.value
-    });
+```js {2-4,7}
+function MyButton() {
+  function handleClick() {
+    alert('You clicked me!');
   }
 
   return (
-    <>
-      <label>
-        First name:
-        <input
-          value={person.firstName}
-          onChange={handleFirstNameChange}
-        />
-      </label>
-      <label>
-        Last name:
-        <input
-          value={person.lastName}
-          onChange={handleLastNameChange}
-        />
-      </label>
-      <label>
-        Email:
-        <input
-          value={person.email}
-          onChange={handleEmailChange}
-        />
-      </label>
-      <p>
-        {person.firstName}{' '}
-        {person.lastName}{' '}
-        ({person.email})
-      </p>
-    </>
+    <button onClick={handleClick}>
+      Click me
+    </button>
   );
 }
 ```
 
-```css
-label { display: block; }
-input { margin-left: 5px; margin-bottom: 5px; }
-```
-
-</Sandpack>
+Notice how `onClick={handleClick}` has no parentheses at the end. You don't need to _call_ your event handler, you need to _pass_ the event handler itself. React will call your handler when the user clicks the button.
```

Comment:
Same here, one of the biggest rookie mistakes is people calling a handler in an `onClick` event, I think if there was some obvious way of highlighting that.

"Do not call the event handler", rather than "You don't need to", because actually you NEED to not do this.

Reply:
I don't think it's possible to over-emphasize this point enough!

(does the `<blink>` tag work inside Markdown? :)  )

Reply:
Yeah I'll try to make this a bit clearer.